content/blog/virtuous-circles.md

```diff
@@ -0,0 +1,92 @@
+---
+author: "Emma Russell"
+date: 2019-09-23
+title: All carrot, no stick - virtuous circles and useful tools
+---
+
+A vicious circle of software use might go something like this: 
+
+*A user employs a suite of generic software tools, together with their own domain knowledge and sense of good practice, 
+to design their own workflow to get a task done. This workflow starts off with the best of intentions, but really only 
+exists in the user's head. Consequently, as time goes on, as deadlines threaten, corners are cut. Next, the cutting of 
+those corners becomes habitual, and the workflow degenerates. Essential outputs are still produced, but steps ensuring 
+transparency and validation may become confused or omitted. Good intentions becomes a distant, guilty memory.* 
+
+The corresponding virtuous circle might be:
+
+*A user employs a software tool which has knowledge of the task in hand, and is structured to encourage good practice. The 
+user is naturally steered towards implementing good practice without having to expend any mental effort on it 
+themselves. As the user's familiarity with the tool increases, doing things 'the right way' becomes second nature and no 
+harder, or actually easier, than doing them the wrong way.* 
+
+The latter might typically involve some higher level tool, with domain knowledge built in. Generic, practice-agnostic 
+software is potentially still employed (e.g. R, Excel, Word) alongside this tool which expresses its domain knowledge to 
+manage the overall workflow.
+
+The virtue of the virtuous circle can be further enhanced by providing additional incentives to users, such as automation 
+of tedious or error-prone sub-tasks, or features for gaining further insight.
+
+A pair of talks in the Reproducible Research stream of this year's [RSE](https://rse.ac.uk/) Conference illustrated this 
+point nicely, and also shared parallels with projects we have developed at RESIDE. 
+
+Both talks addressed the strange fact that, decades into the digital age, scientific papers are still essentially text 
+files, of the sort which Guttenberg could have published on his first printing press. Yet, for papers which rely on 
+computational output, "the paper is advertising, not scholarship", and is fundamentally incomplete without also providing 
+the means for readers to reproduce the computation themselves.
+
+Anna Krystalli of Sheffield University presented [rrtools](https://github.com/benmarwick/rrtools), which enables the 

```

Comment:
Add links to Anna's home page (`https://annakrystalli.me`) and perhaps the sheffield RSE group (`https://rse.shef.ac.uk`)